README.md

```diff
@@ -30,7 +30,8 @@ Create a new collection
 ```python
 client.recreate_collection(
     collection_name="my_collection",
-    vector_size=100
+    vector_size=100,
+    distance="Cosine"
```

Comment:
The tests all use `Distance.COSINE`, the enum spelling. From a user's perspective is it more preferable to use the string or the enum?